### PR TITLE
Refactor/touch ups

### LINKED ISSUE #38: Main page favorite/add-to-cookbook functionality

### LINKED ISSUE #39: Style buttons in the navigation bar

### LINKED ISSUE #40: Change navigation button background color to indicate what view the user is on

### LINKED ISSUE #41: Add hover states (pointer) to main view

### LINKED ISSUE #42: Style the positioning of the favorite/add-to-cookbook buttons on recipe cards

### LINKED ISSUE #43: Add functionality to switch to main view when clicking on 'What's Cookin'

### LINKED ISSUE #44: Delete data files

### PR DESCRIPTION
**Pull request by mr-ryan12**
What (if any) features are you implementing?
- Adjusted some CSS and HTML to enhance the user experience
- Added tests for recipeRepository methods and properties
- Added functionality to switch to home view on button click

What (if anything) did you refactor?
- Refactored HTML, CSS, and JavaScript

Were there any issues that arose?
- Found and fixed multiple bugs when there was unpredicted user behavior

Is there anything that you need from your teammate?
- Code/pull request review.

Any other comments, questions, or concerns?
Next steps will include final refactor and building a README.md

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -41,14 +41,12 @@ <h1>Whats Cookin</h1>
     </nav>
     <main>
       <section class="home-page">
-        <img src="https://spoonacular.com/recipeImages/595736-556x370.jpg" alt="cookies" class="featured-recipe-image">
+        <img src="" alt="cookies" class="featured-recipe-image">
+          <i class="fa fa-heart heart-btn featured-heart-btn"></i>
+          <i class="fa fa-bookmark save-recipe-btn featured-save-recipe-btn"></i>
         <section class="featured-recipe-info-container">
           <h3 class="featured-recipe-info-title">Featured Recipe!</h3>
-          <h4 class="featured-recipe-name">Hot Buttered Cookies!</h4>
```
Comment (philalewis) [inline src/index.html:47]: Got rid of unecessaroy html elements

File: src/scripts.js [hunk 2]
```diff
@@ -231,15 +235,31 @@ function createCookbook() {
...
         <h2 class="recipe-card-title">${recipe.name}</h2>
       </section>`
   });
   addEventListenerToRecipeCards();
   displayYellowBookmarks(user.cookbook);
   displayRedHearts(user.cookbook);
+}
+
+function displayHomeView() {
+  cookbookButton.classList.remove('beige');
```
Comment (philalewis) [inline src/scripts.js:250]: Added a way to get back to the 'home' page, which diplays the recipe of hte day.

File: src/scripts.js [hunk 3]
```diff
@@ -231,15 +235,31 @@ function createCookbook() {
...
+  allRecipesButton.classList.remove('beige');
+  favoritesButton.classList.remove('beige');
+  hide([browsePage, recipeView, cookbook]);
+  show([homePage]);
+}
+
+function updateNavBarButtonColor() {
+  cookbookButton.classList.remove('beige');
```
Comment (philalewis) [inline src/scripts.js:258]: This will let the user know which 'list' of recipes they are filtering through

File: src/styles.css [hunk 4]
```diff
@@ -151,6 +163,18 @@ h1 {
   right: 15%;
 }
 
+.featured-heart-btn {
+  right: 29%;
+}
+
```
Comment (philalewis) [inline src/styles.css:169]: I like the way things are looking on the page right now

**Review by philalewis — APPROVED** [review]
Starting to look really good. We've also gotten rid of a bunch of bugs

File: src/data/users-test-data.js [hunk 5]
```diff
@@ -0,0 +1,149 @@
+const usersData = [
```
Comment (oacortez) [inline src/data/users-test-data.js:1]: Thank you for adding this user data for our testing files

**Review by oacortez — APPROVED** [review]
Thank you for working on this and fixing some styling we discuss. I also see that some of the files we don't need got deleted. Thank you also for adding a testing file for users. Will be merging this!

**Pull request merged**